README.md

```diff
@@ -0,0 +1,288 @@
+                                                          КП ППвИС(часть 1)
+                                                             Установка
+
+Для выполнения первой части КП по курсу ППвИС разработан пример реализации волнового алгоритма поиска минимального пути в графе на базе библиотеки sc-memory.
+
+На вашем компьютере должна быть установлена версия ostis ветки 0.5.0 из репозитория: https://github.com/ShunkevichDV/ostis и Clion.
```

Comment:
Use this link https://github.com/ostis-dev/ostis-web-platform. Update all file with ostis-web-platform insted of ostis